pkg/field/defaults.go

```
@@ -45,9 +62,38 @@
 	logLevelField = StringField("log-level", WithDefaultValue("info"), WithDescription("The log level: debug, info, warn, error"), WithPersistent(true),
 		WithExportTarget(ExportTargetOps))
-	skipFullSync          = BoolField("skip-full-sync", WithDescription("This must be set to skip a full sync"), WithPersistent(true), WithExportTarget(ExportTargetNone))
-	otelCollectorEndpoint = StringField("otel-collector-endpoint", WithDescription("The endpoint of the OpenTelemetry collector to send observability data to"),
+	skipFullSync = BoolField("skip-full-sync", WithDescription("This must be set to skip a full sync"), WithPersistent(true), WithExportTarget(ExportTargetNone))
+
+	// OTEL
+	otelCollectorEndpoint = StringField(OtelCollectorEndpointFieldName, WithDescription("The endpoint of the OpenTelemetry collector to send observability data to (used for both tracing and logging if specific endpoints are not provided)"),
+		WithPersistent(true), WithExportTarget(ExportTargetOps))
+	otelCollectorEndpointTlsCertPath = StringField(OtelCollectorEndpointTLSCertPathFieldName, WithDescription("Path to a file containing a PEM-encoded certificate to use as a CA for TLS connections to the OpenTelemetry collector"),
+		WithPersistent(true), WithExportTarget(ExportTargetOps))
+	otelCollectorEndpointTlsCert = StringField(OtelCollectorEndpointTLSCertFieldName, WithDescription("A PEM-encoded certificate to use as a CA for TLS connections to the OpenTelemetry collector"),
```

golangci-lint (1.64.8)

[error] var-naming: var otelCollectorEndpointTlsCert should be otelCollectorEndpointTLSCert

(revive)